tests/test_pyfepdf.py

```diff
@@ -340,6 +344,8 @@ def test_main_grabar_json():
     f2.close()
     diff = [x for x in d1 if x not in d2]
     assert diff == []
+    os.remove('facturas.txt')
+    os.remove('facturas.json')  
```

Comment:
remove the generated files as they can interfere with later tests when coverage command is run.